monkey/infection_monkey/post_breach/actions/add_user.py

```diff
@@ -1,14 +1,18 @@
 from common.common_consts.post_breach_consts import POST_BREACH_BACKDOOR_USER
 from infection_monkey.config import WormConfiguration
 from infection_monkey.post_breach.pba import PBA
+from infection_monkey.utils.random_password_generator import get_random_password
 from infection_monkey.utils.users import get_commands_to_add_user
 
 
 class BackdoorUser(PBA):
     def __init__(self):
+        remote_user_pwd = get_random_password()
```

Comment:
Would be more readable if called `random_password`, because that's what it is in this context. It's not a password of remote user yet and `pwd` is not consistent (also means print working directory), either `..._pass` or `..._password`